packages/api/src/controllers/stream.js

```diff
@@ -832,6 +847,16 @@ app.put("/:id/setactive", authMiddleware({ anyAdmin: true }), async (req, res) =
       streamId: id,
       userId: user.id,
     });
+  } else {
+    // Trigger stream idle webhook
+    req.queue.emit({
+      id: uuid(),
+      createdAt: Date.now(),
+      channel: "webhooks",
+      event: "stream.idle",
```

Comment:
```suggestion
      event:  req.body.active === true ? "stream.started" : "stream.idle",
```
and remove `if (req.body.active === true)` ?